SwrveCommonSDK/src/main/java/com/swrve/sdk/messaging/SwrveMessageFormat.java

```diff
@@ -41,7 +47,7 @@
      * @return SwrveMessageFormat new instance
      * @throws JSONException
      */
-    public SwrveMessageFormat(SwrveMessage message, JSONObject messageFormatData) throws JSONException {
+    public SwrveMessageFormat(SwrveBase<?, ?> controller, SwrveMessage message, JSONObject messageFormatData) throws JSONException {
```

Comment:
To reduce complexity, I would pass in SwrveConfigBase or ISwrveBase instead of SwrveBase. (Preferably SwrveConfigBase.)